README.md

```diff
@@ -1,6 +1,6 @@
 # Ocular
 
-Ocular seeks to be the preferred cosmos client library UX for Rust projects. It is strongly based on [lens](https://github.com/strangelove-ventures/lens), a go client library for blockchains built with the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
+Ocular seeks to be the preferred cosmos client library UX for Rust projects. It is strongly based on [Ocular](https://github.com/strangelove-ventures/Ocular), a go client library for blockchains built with the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
```

Comment:
this should still say lens, it's based on the go library Jack & stangelove are writing which is called lens